src/styles/sidebar/sidebar.scss

```diff
@@ -60,6 +59,9 @@
 @use './components/Tutorial';
 @use './components/VersionInfo';
 
+// Make the shared package utility styles available
+@use '@hypothesis/frontend-shared/styles/util' as hyp-u;
```

Comment:
Slightly confusingly the `as hyp-u` part doesn't do anything useful here except prevent a clash with the local `util` import. It doesn't affect the exported class names for example.